### PR TITLE
Added PR template and made issue templates ready.

### PR DESCRIPTION
**Pull request by theballmarcus**
I made a PR template and made the other templates ready to be written. I'll fill them in tomorrow.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md/feature_request.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# # # Ready to be writtenn
```
Comment (rodrigo-brito) [inline .github/ISSUE_TEMPLATE.md/feature_request.md:1]: I prefer to not include a empty file, can you remove this file and the bug report?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+##### This PR has:
+
+- [ ] Been doubled checked by the author
+- [ ] Tested for newly introduced bugs
+- [ ] Included documentation
+
+This PR is a <!-- REQUIRED: replace this comment with one of ["small change", "feature", "compatibility breaking update", "non-versioned change"] -->
+that fixes #<!-- replace this comment with an issue number if applicable -->
+
+**Developers**
```
Comment (rodrigo-brito) [inline .github/pull_request_template.md:24]: I prefer to not include this section. Can you remove?

**Review by rodrigo-brito — CHANGES_REQUESTED** [review]
Hi @theballmarcus thank you for the contribution, just a few comments.

**Comment by theballmarcus** [comment]
Thank you for fast responses. I removed the empty files and removed line 21-24.

**Review by rodrigo-brito — APPROVED** [review]
Awesome! Thank you!

**Pull request merged**